ambry-store/src/main/java/com.github.ambry.store/InMemoryIndex.java

```diff
@@ -1,557 +0,0 @@
-/**
```

Comment:
It hasn't been used in a long time. My conjecture is that it was used before `PersistentIndex` was setup and it wasn't removed after. It is no longer used though.